distribution/docker/Dockerfile

```diff
@@ -57,6 +57,7 @@ FROM $BASE_IMAGE
 LABEL maintainer="Apache Druid Developers <dev@druid.apache.org>"
 
 COPY --from=busybox /bin/busybox /busybox/busybox
+USER root
```

Comment:
Why do we need to change to root user?

Reply:
With the new base image, the Dockerfile is built using a non-root user who doesn't have permission to run the command 'rm /busyboxbusybox'. This is why the user is changed. 
Note: this change of user doesn't affect the default user used to log in when we execute the Dockerfile.